Things.md

```diff
@@ -0,0 +1,181 @@
+/* The above is equivalent to the following 2 function calls. */
+setMidX(thing, 1);
+setMidY(thing, 1);
+```
+- Going along with setting midpoints, we can do that based on the midpoint of another Thing. Using `setMidXObj`, `setMidYObj`, and `setMidObj` you can shift a Thing so that its horizontal and/or vertical midpoint(s) are centered on that of another Thing.  
+```javascript
+/* Aligning a Thing's horizontal and vertical midpoints with
+ * that of another Thing. This is equivalent to the following
+ * 2 function calls combined.
+ */
+setMidObj(thing, otherThing);
+
+/* Aligning just the horizontal midpoint. */
+setMidXObj(thing, otherThing);
+
+/* Aligning just the vertical midpoint. */
+setMidYObj(thing, otherThing);
+```
+- The next is to use an `update` function (`updateTop`, `updateBottom`, `updateRight`, or `updateLeft`). This is similar to using both `shift` and `set`, where it will shift the specified bound by the given offset and then update its opposite bound based on height/width.
+```javascript
+/* Updating the top boundary by shifting it up 1. */
+updateTop(thing, 1);
+
+/* Updating the bottom boundary by shifting it up 1.
+ * This is equivalent to the above function.
+ */
+updateBottom(thing, 1);
+
+/* Updating the left boundary by shifting it right 1. */
+updateLeft(thing, 1);
+
+/* Updating the right boundary by shifting if right 1.
+ * This is equivalent to the above function.
+ */
+updateRight(thing, 1);
+```
+- The next is to use `slideToX` or `slideToY`. These will slide a Thing toward a target `x` or `y`, but also limits the total distance allowed (distance computed from the Thing's original midpoint).  
+```javascript
+/* Sliding a Thing to x position 10, moving a max distance of 5. */
+slideToX(thing, 10, 5);
+
+/* Sliding a Thing to y position 10, moving a max distance of 5. */
+slideToY(thing, 10, 5);
+```
+
+## Opacity
+Opacity defines how transparent a Thing is. There is a minimum opacity a Thing can have while not being completely transparent. This is defined in the property `epsilon` and is defaulted to .007. A Thing's opacity is defined in the `opacity` property and ranges from 0 to 1 (0 being completely transparent and 1 being not at all).  
+You can set a Thing's opacity using `setOpacity`.
+```javascript
+/* Setting a Thing's opacity to 1. */
+setOpacity(thing, 1);
+
+/* Setting a Thing's opacity to 0 (transparent). */
+setOpacity(thing, 0);
+```
+You can also set and get a Thing's `epsilon`.
+```javascript
+/* Setting a Thing's epsilon to .005. */
+thing.setEpsilon(.005);
+
+/* Getting a Thing's epsilon. */
+thing.getEpsilon()
+```
+Notice that `setEpsilon` and `getEpsilon` are methods of a Thing, while `setOpacity` is not. This means you need to pass the Thing as an argument to `setOpacity`.
+
+# GroupHoldr
+
+A general storage container for values in keyed Arrays and/or Objects.
+Manipulation utlities are provided for adding, removing, switching, and
+applying methods to values.
+
+
+## Basic Architecture
```

Comment:
By copy&paste I didn't realize you meant the entire readme :( that's not good.

Give examples similar to what's above.